framework/src/meshgenerators/XYDelaunayGenerator.C

```diff
@@ -163,6 +173,22 @@ XYDelaunayGenerator::XYDelaunayGenerator(const InputParameters & parameters)
   for (auto hole_i : index_range(_stitch_holes))
     if (_stitch_holes[hole_i] && (hole_i >= _refine_holes.size() || _refine_holes[hole_i]))
       paramError("refine_holes", "Disable auto refine of any hole boundary to be stitched.");
+
```

Comment:
Check if both `interior_points` and `interior_point_files` are specified. If so, throw an error.

Initialize `_interior_points` here based on the input.

Reply:
Why should this error? The way the code is setup is that a user can provide 1 or more files and/or provide points in the input files. The set of interior points will be the union of all these points.

Reply:
Oh, I was thinking one position file or direct `interior_points`, which is the approach used in other similar scenarios.

Do we have any preference here @GiudGiud?

Reply:
@GiudGiud What do you think?

Reply:
allowing the dual input is fine by me as long as it works
ideally we also should detect when the user is specifying the same hold with both inputs

Reply:
I have an example where we can use both for specifying the interior points. If a user specifies the same point twice, the code already throws an error.

Reply:
Is the error thrown by MOOSE, libmesh or poly2tri? A moose error that guides users to the specific MG/input parameters would be appreciated. Anyway, IMO we need a test for this error.